app/src/features/Incentives/components/IncentiveTasksList/IncentiveTasksList.tsx

```diff
@@ -0,0 +1,66 @@
+import { Collapse } from "antd";
+import { CreditsProgressBar } from "../CreditsProgressbar/CreditsProgessbar";
+import { IncentiveSectionHeader } from "../IncentiveSectionHeader";
+import { incentiveTasksList } from "./constants/incentiveTasks";
+import { PiCaretDownBold } from "@react-icons/all-files/pi/PiCaretDownBold";
+import { MdCheckCircle } from "@react-icons/all-files/md/MdCheckCircle";
+import { IncentiveTaskListItem } from "./types";
+import { useNavigate } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import "./incentiveTasksList.scss";
+
+export const IncentiveTasksList = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const renderTaskHeader = (task: IncentiveTaskListItem) => {
```

Comment:
Avoid creating renderSomething functions
```suggestion
  const taskHeader = <>...</>
```